test/all.bash

```diff
@@ -122,9 +120,7 @@ for dir in unit lib; do
     td=$VIRGIL_LOC/test/$dir
     print_line
     echo "${CYAN}($V3C_STABLE) $dir${NORM}"
-    (cd $td && AENEAS_TEST=$V3C_STABLE $td/test.bash)
-    X=$?
-    update_exit_code_if_non_zero $X
+    (cd $td && AENEAS_TEST=$V3C_STABLE $td/test.bash) || exit_if_non_zero $?
```

Comment:
I think you can just do `|| exit $?`

Reply:
Oh right bash has shortcircuiting conditionals. Done :+1: